docs/transparent-validation/muxpi/theory-of-operations.md

```diff
@@ -33,13 +33,15 @@ RX`, `DBG UART TX` on pinout image below):
 ![](../../images/muxPi-addon-left.png)
 
 and plug USB connector to the computer. Next, open connection in PC terminal by
-typing: `sudo minicom -D /dev/ttyUSB0 -o -b 115200`.
+typing:
+
+    sudo minicom -D /dev/ttyUSB0 -o -b 115200.
```

Comment:
Code blocks should be added as follows:

```bash
<insert Your code here>
```
Please refer to another files in documentation.